src/server/route/routes/node.ts

```diff
@@ -1,9 +1,26 @@
 import { router } from '../router';
 import { Request, Response } from 'express';
 import { INode } from '../../../../types';
-//import {IError} from '../../domain/IError';
 import { db } from '../../dbConfigs';
 
+// Checks need to make sure the node is valid
+const nodeCheck = (node: INode): boolean => {
+    //Check that the node has all properties
+    if (
+        node.label &&
+        node.status &&
+        node.priority &&
+        // eslint-disable-next-line no-prototype-builtins
+        node.hasOwnProperty('x') &&
+        // eslint-disable-next-line no-prototype-builtins
+        node.hasOwnProperty('y')
+    ) {
+        return true;
+    }
+
+    return false;
```

Comment:
instead of `if (X) { return true; } else { return false; }` just do `return X;`

Reply:
I wouldn't leave it like that if I didn't have problems with syntax. I guess I will continue trying.

Reply:
you can type cast it with `Boolean(X)`